servers/cromwell/README.md

```diff
@@ -28,7 +28,13 @@ latter lists only direct dependencies. To regenerate run:
 ```
 virtualenv --python=/usr/bin/python2 /tmp/cromwell-server-requirements
 source /tmp/cromwell-server-requirements/bin/activate
-pip install -r servers/cromwell/requirements-to-freeze.txt
-pip freeze | sort > servers/cromwell/requirements.txt
+```
+Then, from the cromwell directory in this repo:
+```
+pip install -r requirements-to-freeze.txt
+pip freeze | sed 's/^jm-utils.*/\.\.\/jm_utils/g' > requirements.txt
 deactivate
 ```
+
+The sed command above replaces jm-utils=x.y.z with ../jm_utils, which is required
```

Comment:
This is unfortunate, but I don't have a better suggestion. Perhaps @bfcrampton has an idea.

Reply:
Yeah, I don't really like it either. Long term, I think a better way to share code would be to make `servers` one big installable Python package. That way we wouldn't need to put jm_utils into requirements.txt at all. I started exploring that but it requires big changes to the docker compose files, and needs more discussion. Assuming people want to go that way, I think it's more appropriate to make a separate ticket and PR to do it later.

Reply:
@dshiga for now I liked your suggestion to explicitly install `jm_utils` in the dockerfile instead of putting it in `requirements.txt`, because then we wouldn't need to do this sed command to fix the generated requirements

Reply:
It seemed attractive to just copy the jm_utils directory into the docker image, instead of pip installing it, but it doesn't mesh well with tox, which wants all dependencies installed via requirements files. We could put jm_utils in cromwell's and dsub's test-requirements.txt to get around that. I think it would work, but it feels risky to have dependencies installed one way in the Dockerfiles and another for unit tests. What do you think?

Reply:
That's a good point! As discussed in person, I think it's better to leave it as-is in this case for consistency.

Reply:
I looked into it a bit and couldn't come up with any better solutions. I agree the way it is now is probably best for consistency.